src/components/AudioCell.vue

```diff
@@ -275,6 +295,20 @@ const isMultipleEngine = computed(() => store.state.engineIds.length > 1);
     height: 2rem;
   }
 
+  .line-number {
+    height: 2rem;
+    width: v-bind(lineNumberWidth);
+    line-height: 2rem;
+    opacity: 0.6;
+    text-align: right;
+    color: colors.$display;
+    &.active {
+      opacity: 1;
+      font-weight: bold;
+      color: colors.$primary;
```

Comment:
```suggestion
      color: colors.$primary-light;
```
左の三角と同じ色にしておくとカスタムテーマなどを作るときにやりやすそうな感じがしました。

Reply:
Suggestion ありがとうございます！
そうですね！　視認性もさほど変わらないので, 左の緑の三角の色に揃えたいと思います